src/appsco-app.html

```diff
@@ -3998,6 +4022,10 @@
                 deleteContact.toggle();
             },
 
+            _onDeleteCustomerAction: function(event) {
```

Comment:
No error. Just an instruction: "TODO - description" should be used for methods that are going to be implemented.